src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -11,8 +11,8 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.Operations
     Partial Friend NotInheritable Class VisualBasicOperationFactory
-        Private Shared Function ConvertToOptional(value As ConstantValue) As [Optional](Of Object)
-            Return If(value Is Nothing OrElse value.IsBad, New [Optional](Of Object)(), New [Optional](Of Object)(value.Value))
+        Private Shared Function ConvertToOptional(value As ConstantValue) As OperationConstantValue
```

Comment:
>ConvertToOptional [](start = 32, length = 17)

I think this helper was renamed on C# side. BTW, can we share is between languages?

Reply:
Removed the methods entirely.

---
In reply to: [444579977](https://github.com/dotnet/roslyn/pull/45127#discussion_r444579977) [](ancestors = 444579977)